src/lib/Event/Subscriber/RoutingListener.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\CompatibilityLayer\Event\Subscriber;
+
+use Ibexa\Bundle\Core\EventListener\RoutingListener as CoreRoutingListener;
+
+class RoutingListener extends CoreRoutingListener
```

Comment:
It's a bit risky, because there's no BC for extending listeners, unless specifically documented. But maybe +1 given it's just the compatibility layer.